samples/AspNetCore/MyWebhookEventProcessor.cs

```diff
@@ -6,22 +6,18 @@
 using Octokit.Webhooks.Events;
 using Octokit.Webhooks.Events.PullRequest;
 
-public class MyWebhookEventProcessor : WebhookEventProcessor
+public class MyWebhookEventProcessor(ILogger<MyWebhookEventProcessor> logger) : WebhookEventProcessor
```

Comment:
Are these changes related to the update?

Reply:
Yeah, it's a new feature called [primary constructors][1]. Because we enable all warnings, and warnings as errors, the new SDK brings in [IDE0290][2] which forces us to adopt it.

It doesn't impact downstream consumers, as this is still compiled to the same CIL.

[1]: https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/primary-constructors
[2]: https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/style-rules/ide0290